toaster/__init__.py

```diff
@@ -1,55 +1,46 @@
 import os, requests
 import datetime
-
+import json
 from toaster.helper import get_config_path, load_default_id, format_time
 
-USERID = load_default_id()
-API_ENDPOINT = 'https://us-central1-toaster-253815.cloudfunctions.net/toaster_message'
+webhook_url = load_default_id()
 
 def toast(method):
```

Comment:
I'm thinking maybe you can create a new function called `slack_toast`, you may reference from `telegram_toast` function.

Reply:
https://github.com/zhunhung/toaster/pull/2/files

you can add me to the contributors, please :)